Library/Formula/cdrtools.rb

```diff
@@ -45,6 +60,10 @@ def install
 
   test do
     system "#{bin}/cdrecord", "-version"
+    system "#{bin}/cdda2wav", "-version"
```

Comment:
New bottles won't be created unless the person who merges this chooses to (which they probably won't)